### ISSUE TITLE
Hey man, where is the documentation? You promised. :(

### ISSUE THREAD

**Issue by pauloops** [thegrubbsian/jquery.ganttView#24]

**Comment by msm2020** [comment]
:( 
where is the documentation ?

**Comment by lutze** [comment]
It'll be coming over the next 2-ish months, along with hopefully some of the cool new features people have been adding to their forks over the last couple years.

**Comment by brylie** [comment]
Agreed, a little documentation goes a long way. Documentation is important, so we know how to use this package. Any ETA on **basic** documentation?
